satellite/gc/config.go

```diff
@@ -0,0 +1,16 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package gc
+
+import "time"
+
+// Config contains configurable values for garbage collection
+type Config struct {
+	Interval time.Duration `help:"how frequently garbage collection filters should be sent to storage nodes" releaseDefault:"168h" devDefault:"168h"`
+	Active   bool          `help:"set if garbage collection is actively running or not" releaseDefault:"true" devDefault:"true"`
+
+	// TODO: find out what initial number of pieces should be when creating a filter
+	InitialPieces     int64   `help:"the initial number of pieces expected for a storage node to have, used for creating a filter" releaseDefault:"" devDefault:""`
```

Comment:
This should come from the last run or some other place. This value is different for each storagenode.

Reply:
yeah but if garbage collection is starting up for the first time and there is no last run, then it needs this initial starting value for pieces.

Reply:
updated to use the config value (average number of pieces per storagenode - approx. 400k) for the first run of garbage collection. The first run tracks the number of pieces _actually_ on each storagenode, and those values are used in the next run instead of the config value